### DISCUSSION TITLE
cloud folders

### DISCUSSION THREAD

**Discussion by ttemplin99** [Geektoolkit/Dynaframe3#69] [General]
hi there, 

pretty new to the pi world but in one of the last dyna frame youtube videos there was mention of onedrive support. how do i set this up? if i can have a onedrive folder for my photos that can be added to at anytime and from anywhere that would be great. 

thanks

**Comment by Geektoolkit** [comment]
You'd use Rclone for that. https://rclone.org/downloads/.  Rclone essentially syncs your cloud (onedrive/dropbox,etc) to the Pi, and then dynaframe can load the images from there.  I'd look at rclone tutorials for now to get it to sync from a onedrive folder to your Pictures folder for the easiest setup.  I have plans in my backlog of videos to document this, but it's a little bit back as i'm catching up after losing a hard drive
